supplements/sword-coast-adventurers-guide/classes/sorcerer-stormsorcery.xml

```diff
@@ -83,13 +88,14 @@
 			<description>When you are hit by a melee attack, you can use your reaction to deal {{level:sorcerer}} lightning damage to the attacker. The attacker must also make a DC{{spellcasting:dc:cha}} Strength saving throw. On a failed save, the attacker is pushed in a straight line up to 20 feet away from you.</description>
 		</sheet>
 	</element>
+
 	<element name="Wind Soul" type="Archetype Feature" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_ARCHETYPE_FEATURE_STORM_SORCERY_WIND_SOUL">
 		<description>
 			<p>At 18th level, you gain immunity to lightning and thunder damage.</p>
 			<p class="indent">You also gain a magical flying speed of 60 feet. As an action, you can reduce your flying speed to 30 feet for 1 hour and choose a number of creatures within 30 feet of you equal to 3 + your Charisma modifier. The chosen creatures gain a magical flying speed of 30 feet for 1 hour. Once you reduce your flying speed in this way, you can't do so again until you finish a short or long rest.</p>
 		</description>
-		<sheet>
-			<description>You gain a magical flying speed of 60 feet. As an action, you can reduce your flying speed to 30 feet for 1 hour and choose {{wind soul:creatures}} creatures within 30 feet of you that gain a magical flying speed of 30 feet for 1 hour. (1/LR)</description>
+		<sheet action="Action" usage="1/Short Rest">
```

Comment:
Is there precedent for "1/Short Rest" or "1/Short or Long Rest"

Reply:
Not really, every "per Short Rest" is also a "per Long Rest"